packages/lesswrong/components/community/Community.tsx

```diff
@@ -213,7 +211,16 @@ const Community = ({classes}: {
     // unfortunately the hash is unavailable on the server, so we check it here instead
     if (location.hash === '#online') {
       setTab('online')
+    } else if (location.hash === '#individuals') {
+      setTab('individuals')
+    }
+    
+    // only US and UK default to miles - everyone else defaults to km
+    // (this is checked here to allow SSR to work properly)
+    if (['en-US', 'en-GB'].some(lang => lang === window?.navigator?.language)) {
+      setDistanceUnit('mi')
```

Comment:
Moved this to the parent component so that it doesn't get reset every time you change tabs